security/fma-interop-tx-handling.md

```diff
@@ -0,0 +1,137 @@
+    the Supervisor FMA.
+    - Negative UX and Customer Perception from building invalid block content.
+
+## FM2: Checks Discard Valid Message
+- Description
+    - Due to a bug in either the Supervisor, or the Callers, some or all Executing Messages
+    aren't being included in blocks.
+    - When this happens, there is nothing invalid being produced by block builders, but no Interop
+    Messages are being included.
+- Risk Assessment
+    - More Negative UX and Custoemr Perception if Interop Messages aren't making it into the chain.
+    - Failed transactions would cause customers to redrive transactions, potentially overwhelming
+    infrastructure capacity.
+
+## FM3a: Transaction Volume causes DOS Failures of Proxyd
+- Description
+    - Due to the new validation requirements on `proxyd` to check Interop Messages, an influx of
+    Interop Messages may arrive, causing `proxyd` to become overwhelmed with work.
+    - When `proxyd` becomes overwhelmed, it may reject customer requests or crash outright, affecting
+    liveness for Tx Inclusion and RPC.
+- Risk Assessment
+    - Low Impact ; Medium Likelihood
+    - If a `proxyd` instance should go down, we should be able to replace it quickly, as it is a stateless
+    service.
+    - When `proxyd` goes down, it takes outstanding requests with it, meaning some requests fail to be handled,
+    but excess load is shed.
+- Mitigations
+    - `proxyd` could feature a pressure-relief setting, where if too much time is spent waiting on the Supervisor,
+    no additional Interop Messages will be accepted through this gateway.
+    - We should deploy at least one "Utility Supervisor" to respond to Checks from `proxyd` instances.
+    The size and quantitiy of the Supervisor(s) could be scaled if needed. (Note: A Supervisor also requires Nodes
+    of each network to function)
+## FM3b: Transaction Volume causes DOS Failures of Supervisor
+- Description
+    - In any section of infrastructure, calls to the Supervisor to Check a given Interop Message might overwhelm the
+    Supervisor.
+    - If this happens, the Supervisor may become slow to respond, slow to perform its other sync duties, or may crash outright.
+    - When a Supervisor crashes, any connected Nodes can't keep their Cross-Heads up to date, and Managed Nodes won't get L1 updates either.
+- Risk Assessment
+    - Medium Impact ; Low Likelihood
+    - The Supervisor is designed to respond to Check requests. Even though it hasn't been load tested in realistic settings, there is very little computational overhead when responding to an RPC request.
+    - Supervisors can be scaled and replicated to serve high-need sections of the infrastructure. Supervisors
+    sync identically (assuming a matching L1), so two of them should be able to share traffic.
+- Mitigations
+    - When the Supervisor is down, any block builder or mempool filter *should* treat unavailability as
```

Comment:
The liveness of cross chain messages is decoupled from the liveness of the chain. Its possible to keep the chain running without the ability to process cross chain messages when the supervisor goes down

Reply:
Right, that's what I'm trying to say here - when block builders can't assess Interop Validity (by way of not having a Supervisor), they can choose to drop interop messages. In this way, the liveness of interop is managed separately to preserve the chain liveness.

I'll do a pass on this content to make sure its closer to your description.